index.js

```diff
@@ -123,13 +129,22 @@ function createQuestions(config) {
  * @param {Object} answers Answers provide by `inquier.js`
  * @return {String} Formated git commit message
  */
-function format(answers) {
+function format(answers, config) {
+  const format = config.format || '{type} {scope} {subject}'
```

Comment:
I'd consider making the default format `'{type} ({scope}) {subject}'`. This way if someone wants to change how scope is decorated, they can using the `config.format` option.

This also simplifies `  const scope = answers.scope ? '(' + answers.scope.trim() + ') ' : ''` on the following line.

Reply:
@Jakobo  I can incorporate those changes, provided @ngryman is willing to merge this

Reply:
@mohamedfasil I will 👍 (cf. PR comment).

I agree with @Jakobo , we're changing the default behavior here. The default should keep being: `{type} ({scope}) {subject}`